tests/store.js

```diff
@@ -71,7 +71,9 @@ suite['should not store default args'] = function (test) {
   ]
 
   ctrl.services({
-    utils: 'test'
+    utils: function () {
```

Comment:
Hm, I think the main test suite above covers the complexity of service registration. Not really sure what this test actually does? I think we should have a test review when moving to 0.34, as we have to go through all tests anyway to change usage of signals vs addSignals and so on.